modules/vba_documents/app/workers/vba_documents/report_unsuccessful_submissions.rb

```diff
@@ -30,20 +29,66 @@ def stuck
       ).order(:consumer_name, :status)
     end
 
+    # rubocop:disable Metrics/MethodLength
+    # rubocop:disable Metrics/PerceivedComplexity
+    # rubocop:disable Metrics/CyclomaticComplexity
+    # rubocop:disable Metrics/AbcSize
     def totals
```

Comment:
This method is definitely due for refactoring. I think disabling the cops here should be done in [the todo file](https://github.com/department-of-veterans-affairs/vets-api/blob/master/.rubocop_todo.yml) rather than inline. Disabling inline/in file is equivalent to saying "Rubocop was wrong here and this code is how we want it."